civicrm.module

```diff
@@ -1046,6 +1050,30 @@ function civicrm_preprocess_page(&$variables) {
   if ($region = CRM_Core_Region::instance('html-header', FALSE)) {
     CRM_Utils_System::addHTMLHead($region->render(''));
   }
+  // to increase it's flexibility.
+  if (module_exists('date_popup') && (in_array(arg(0), array('civicrm', 'user')))) {
```

Comment:
`data_popup` doesn't exist in Backdrop. It now uses HTML5 date element so I'm not sure if we need this at all. A good chunk of this is commented out.